core/db/migrate/20160420181916_migrate_credit_cards_to_wallet_payment_sources.rb

```diff
@@ -19,7 +27,7 @@ def down
     add_column :spree_credit_cards, :default, :boolean, default: false, null: false
 
     wallet_payment_sources = Spree::WalletPaymentSource.
-      where(default: true, source_type: 'Spree::CreditCard').
+      where(default: true, payment_source_type: 'Spree::CreditCard').
       includes(:source)
```

Comment:
we'll need some tweaking for the `includes` to work with the in-migration classes right?